app/konnect/servicehub/service-documentation.md

```diff
@@ -87,18 +91,19 @@ Deleted files are permanently removed from the Dev Portal.
 
 ### Publish a service {#publish}
 
-Publish a service and its API specs to the Dev Portal. Publishing a service makes it available to developers in your organization.
+Publish a service and its API specs to the Dev Portal. Publishing a service makes it available to developers in your organization. Unpublishing a service removes it from the Dev Portal. Removing a service from the Dev Portal is not permanent, and you can republish it at any time.
 
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
-Publish the service:
+{:.note}
+> **Note**: You can only publish services in your [geographic region](/konnect/regions) to the Dev Portal in your region. If you want to publish services to a Dev Portal in another region, switch to the new region in the top-right of {{site.konnect_product_name}}.
 
-* From the **Service actions** drop-down menu, select **Publish to portal**.
+From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service. 
+Then use the **Service actions** drop-down menu to select **Publish to portal** or **Unpublish from portal**.
 
-### Unpublish a service {#unpublish}
 
-Unpublish a service to remove it from the Dev Portal. Removing a service from the Dev Portal is not permanent, and you can republish it at any time.
+### Publishing Service Documentation
 
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
-Unpublish the service:
+Publishing and unpublishing service documentation in {{site.konnect_product_name}} controls what document is displayed for a particular service.
+Publishing is managed from a specific service's **Documentation Page**. 
 
-* From the **Service actions** drop-down menu, select **Unpublish from portal**.
+From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, and click the **Documentation** page. 
+From the **Documentation page**, click the context menu for a specific service document, select **edit**, and toggle the **published page** button to the desired state.
```

Comment:
```suggestion
1. From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, and click the **Documentation** page. 

1. From the **Documentation** page, click the context menu for a specific service document, select **edit**, and toggle the **published page** button to the desired state.
```

Reply:
I'm reminded that we decided to include that preamble as a step going forward, in one of our earlier style discussions with Diana. That is, the "From the service hub... etc" instruction.